Automattic-Tracks-iOS/TracksConstants.m

```diff
@@ -1,4 +1,4 @@
 #import "TracksConstants.h"
 
 NSString *const TracksErrorDomain = @"TracksErrorDomain";
-NSString *const TracksLibraryVersion = @"0.7.1";
+NSString *const TracksLibraryVersion = @"0.8.0";
```

Comment:
Wow was about to say you should update the `podspec` too but I just saw that it actually already [parses that `.m` file dynamically](https://github.com/Automattic/Automattic-Tracks-iOS/blob/develop/Automattic-Tracks-iOS.podspec#L3), nice. Not sure how stable that trick/solution is (e.g. what if we `pod trunk push` while the PWD is not the repo root, not sure the relative path used by `File.read` will work? etc) but something separate to a release cut, so might inspect that separately.

Reply:
Yeah, I really don't know how I feel about the whole parsing-the-version thing in this repo 😅

Reply:
I would not blame you if you decide to remove that parsing part and switch to hardcoded version before merging I you want to 😇